src/components/Dashboard.js

```diff
@@ -4,487 +4,484 @@ class Dashboard extends Component {
-                                                          </label>
+                                                        <label>
+                                                          <input type="checkbox" name="optionsCheckboxes" /><span className="checkbox-material"><span className="check"></span></span>
+                                                        </label>
                                                       </div>
-                                                  </td>
-                                                  <td>Lines From Great Russian Literature? Or E-mails From My Boss?</td>
-                                                  <td className="td-actions text-right">
+                                                    </td>
+                                                    <td>Lines From Great Russian Literature? Or E-mails From My Boss?</td>
+                                                    <td className="td-actions text-right">
                                                       <button type="button" rel="tooltip" title="" className="btn btn-primary btn-simple btn-xs" data-original-title="Edit Task">
-                                                          <i className="material-icons">edit</i>
+                                                        <i className="material-icons">edit</i>
                                                       </button>
                                                       <button type="button" rel="tooltip" title="" className="btn btn-danger btn-simple btn-xs" data-original-title="Remove">
-                                                          <i className="material-icons">close</i>
+                                                        <i className="material-icons">close</i>
                                                       </button>
-                                                  </td>
-                                              </tr>
-                                              <tr>
-                                                  <td>
+                                                    </td>
+                                                  </tr>
+                                                  <tr>
+                                                    <td>
                                                       <div className="checkbox">
-                                                          <label>
-                                                              <input type="checkbox" name="optionsCheckboxes" checked="" /><span className="checkbox-material"><span className="check"></span></span>
-                                                          </label>
+                                                        <label>
+                                                          <input type="checkbox" name="optionsCheckboxes" defaultChecked /><span className="checkbox-material"><span className="check"></span></span>
```

Comment:
I think for our checkbox, it's good idea to have a default setting, we can use defaultChecked...

Reply:
Once these are connected to the assessment model's answers, we won't need to do this.  In order for them to look like some are checked right now I had to change to this. React doesn't like it when the checkboxes are manually checked.  That's why they make you use the defaultChecked prop.